src/builder/cfg.rs

```diff
@@ -25,24 +25,28 @@ pub struct CFGBuilder<T> {
     pub(super) n_out_wires: usize,
 }
 
-impl<B: AsMut<Hugr> + AsRef<Hugr>> Container for CFGBuilder<B> {
+impl<B: Buildable> Buildable for CFGBuilder<B> {
```

Comment:
Indeed, if/presuming that works, one could have a macro implementing `HugrMut` for all of these, which would allow combining `Buildable` into `HugrMut`. (If we want `HugrMut`-ness spread so widely).